New text on Markdown in 02-make-it-public.md (#104)
SoftDev4Research/4OSS-lesson: _episodes/02-make-it-public.md

    - This file is automatically displayed on the front page of your repository
    - It has important information about your project

>  The '.md' extension indicates that the file is written in Markdown language, which uses plain text formatting syntax that can be converted into web output format. Markdown syntax is basically plain text with a few non-alphabetic characters such as * or # for bold, italics, headings, etc. to style test. For example, you can make some text bold by adding two asterisks \** before and after the text: 

Reviewer: Small fix: 

> to style test

maybe should be 

> to style text

Reviewer 2: Thank you @dbampalikis for your contribution. I like the more description of makdown, but I don't agree with removing the links to the guides. Can you merge those two?

Reviewer: @mkuzak The pull request is from @allegravia , I only added a comment about a misspelled word.

Reviewer 2: yes, you're right, thank you for reviewing :)

Reviewer 2: > Thank you @dbampalikis for your contribution. I like the more description of makdown, but I don't agree with removing the links to the guides. Can you merge those two?

@allegravia can you make this change?

 
```
**This text is bold** 
```
will appear as: 

**This text is bold**
 
It is also possible to create lists, insert images, tables, web links and more.
 
We will see now some basic writing and formatting syntax by following this [short guide](https://guides.github.com/features/mastering-markdown/). This [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) also provides useful guidance. 
 
> ## Challenge: Improve the README
>
> The README.md file that GitHub generated when we created the project is quite
> boring. Add some more content using some of the Markdown syntax introduced
> above (if you don't remember a Markdown command you can refer to the 
> [cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) mentioned above). 
> You could add a few sentences about the purpose of your software, some
> details of the author(s) and their affiliation(s), or anything else that you
> think is important.

{: .challenge}

> In addition to the README.md file, a good software project should include other "recommended community health files". What other files should you include in your project? You may find insights on health files to include in your project (to ensure that it meets the recommended community standards) in the Community Profile of the public repository hosting your project. Indeed, a public repository Community Profile provides rules and best practices on how to help grow a community and support contributors in using and contributing to your project. If your project is hosted on GitHub, you may refer to the [GitHub Community Profile](https://github.com/SoftDev4Research/4OSS-lesson/community), which can be accessed following [these instructions](https://help.github.com/en/articles/accessing-a-projects-community-profile) and where you can find a checklist that can be used to see if your project includes community health files recommended by GitHub. 

Reviewer 2: ```suggestion
In addition to the README.md file, a good software project should include other "recommended community health files". What other files should you include in your project? You may find insights on health files to include in your project (to ensure that it meets the recommended community standards) in the [Community Profile](https://help.github.com/en/articles/about-community-profiles-for-public-repositories) of the public repository hosting your project. Indeed, a public repository Community Profile provides rules and best practices on how to help grow a community and support contributors in using and contributing to your project. If your project is hosted on GitHub, you may refer to the [GitHub Community Profile](https://github.com/SoftDev4Research/4OSS-lesson/community), which can be accessed following [these instructions](https://help.github.com/en/articles/accessing-a-projects-community-profile) and where you can find a checklist that can be used to see if your project includes community health files recommended by GitHub. 
```

Reviewer 2: ```suggestion
In addition to the README.md file, a good software project should include other [recommended community health files](04-contributions.md). What other files should you include in your project? You may find insights on health files to include in your project (to ensure that it meets the recommended community standards) in the Community Profile of the public repository hosting your project. Indeed, a public repository Community Profile provides rules and best practices on how to help grow a community and support contributors in using and contributing to your project. If your project is hosted on GitHub, you may refer to the [GitHub Community Profile](community.md), which can be accessed following [these instructions](https://help.github.com/en/articles/accessing-a-projects-community-profile) and where you can find a checklist that can be used to see if your project includes community health files recommended by GitHub. 
```

> These include a project DESCRIPTION file, a README file and a CODE_OF_CONDUCT, which contains standards on how to engage in the community. Furthermore, a project should have a CONTRIBUTING file (for details see episode on "Define clear and transparent contribution, governance and communication processes") outlining guidance for contributing to the project, and a LICENCE file that states how the code can be used. You should have a good idea of what is the appropriate license for your project after the [next episode](https://softdev4research.github.io/4OSS-lesson/03-use-license/index.html). Templates for GitHub issues and pull requests are also required for a complete community profile (see the [Software Carpentry lesson on Version Control with Git](http://swcarpentry.github.io/git-novice/) to learn about GitHub issues and pull requests).

Reviewer 2: ```suggestion
These include a project DESCRIPTION file, a README file and a CODE_OF_CONDUCT, which contains standards on how to engage in the community. Furthermore, a project should have a CONTRIBUTING file (for details see episode on "Define clear and transparent contribution, governance and communication processes") outlining guidance for contributing to the project, and a LICENCE file that states how the code can be used. You should have a good idea of what is the appropriate license for your project after the [next episode](03-use-license.md). Templates for GitHub issues and pull requests are also required for a complete community profile (see the [Software Carpentry lesson on Version Control with Git](http://swcarpentry.github.io/git-novice/) to learn about GitHub issues and pull requests).
```

> You might also want to add details of how to correctly cite the software after publication. This can be done by adding a file called CITATION.cff in the root of your repository (where other health files are stored). The .cff (Citation File Format) is a machine-readble and human-readble and -writable format providing citation information for research software.

Reviewer 2: ```suggestion
You should also add description on how others can correctly cite the software. This can be done by adding a file called CITATION.cff in the root of your repository (where other health files are stored). The .cff (Citation File Format) is a machine-readble and human-readble and -writable format providing citation information for research software.
```
